.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] [OPTIONAL] Hi you can add this file to your .gitignore as it is a vscode configuration file and doesn't need to be committed to the repo